packages/gatsby-theme-aio/algolia/query-builder.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const AlgoliaHTMLExtractor = require('algolia-html-extractor');
+
+class QueryBuilder {
+  constructor () {
+    this.htmlExtractor = new AlgoliaHTMLExtractor();
+  }
+
+  /**
+   * @return {Array}
+   */
+  build = (options = {}) => {
+    this.cssSelector = options.cssSelector ? options.cssSelector : 'p,li,td,code';
+    const mdxQuery = options.mdxQuery ? options.mdxQuery : `
```

Comment:
```suggestion
    const mdQuery = options.mdQuery ? options.mdQuery : `
```

You can change the query name to `mdQuery` because it is not a mdxQuery. We are using the `allMarkdownRemark` query from the `gatsby-transformer-remark` plugin to get the html, so we're just pulling from plain markdown.

Reply:
> So we're just pulling from plain markdown.

Is transcluding content pulled in this case ? See https://github.com/adobe/gatsby-theme-aio#embedding-markdown-documents-and-filtering-content

UXP is a strong user of transcluded content eg https://github.com/AdobeDocs/uxp-photoshop/search?q=import

Reply:
@icaraps 
Looks like no, transcluding content doesn't pulled to index.
I will create a separate ticket to support it.